src/redux/search/actionCreater.js

```diff
@@ -0,0 +1,49 @@
+import { API_KEY, API_URL } from 'react-native-dotenv';
```

Comment:
In redux we call these files actionCreat**o**r-s instead of creater, pls rename it.